org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -697,3 +697,51 @@ GithubOidcSageBionetworksMTBSageIT:
   DefaultOrganizationBinding:
     Account: !Ref SageITAccount
     Region: us-east-1
+
+# for https://github.com/Sage-Bionetworks/synapse-login-aws-infra
+GithubOidcSageBioNetworksScSynapseLoginInfra:
```

Comment:
this setup already exists at https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/650-identity-providers/_tasks.yaml#L443-L501

just need to add appropriate accounts

Reply:
fixed